Randominator/UI/Pages/Generators/BookTitle.razor

```diff
@@ -0,0 +1,66 @@
+﻿@page "/generators/Book-Title"
```

Comment:
Let's use all-lowercase for path, to keep the things consistent.

Reply:
o7